app.go

```diff
@@ -98,12 +98,65 @@ type App struct {
 	ready chan struct{}
 }
 
-// NewWindow creates a window that will oad the given URL and will display
+// WindowOptions contains options for creating windows
+type WindowOptions struct {
+	URL              string // Initial URL to load; leave empty to load nothing
```

Comment:
If it should be an error, I would probably change `OpenWindow`'s signature to start with `url string`.